workers/common/src/test/resources/.ort.env.direct-variables.yml

```diff
@@ -0,0 +1,5 @@
+environmentVariables:
+- name: "variable1"
+  value: "testSecret1"
```

Comment:
Nit: Maybe do not call the values "testSecret". The point of direct variable values is that they are not sensitive.